### PR TITLE
Browser Documentation and Fixes

### PR DESCRIPTION
**Pull request by MicaiahReid**
Expands use of `Error.captureStackTrace` to work in firefox.

Updates README with info about using Ganache in the browser.

### CODE REVIEW

**Review by davidmurdoch — APPROVED** [review]
Awesome. Yay!

⭐⭐⭐⭐⭐

File: src/packages/ganache/README.md [hunk 1]
```diff
@@ -192,6 +192,23 @@ const ganache = require("ganache");
...
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/ganache@version"></script>
+```
+
+NOTE: the `version` in the above path needs to be replaced with a version number or tag that is listed in [npm](https://www.npmjs.com/package/ganache?activeTab=versions).
+
+From there, Ganache is available in your browser for use:
+
+```Javascript
```
Comment (davidmurdoch) [inline src/packages/ganache/README.md]:
```suggestion
```javascript
```

File: src/packages/ganache/README.md [hunk 2]
```diff
@@ -192,6 +192,23 @@ const ganache = require("ganache");
...
 
+### Browser Use
+
+You can also use Ganache in the browser by adding the following script to your HTML:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/ganache@version"></script>
+```
+
+NOTE: the `version` in the above path needs to be replaced with a version number or tag that is listed in [npm](https://www.npmjs.com/package/ganache?activeTab=versions).
```
Comment (MicaiahReid) [inline src/packages/ganache/README.md]:
```suggestion
NOTE: the `{VERSION}` in the above path needs to be replaced with a version number or tag that is listed in [npm](https://www.npmjs.com/package/ganache?activeTab=versions).
```

File: src/packages/ganache/README.md [hunk 3]
```diff
@@ -192,6 +192,23 @@ const ganache = require("ganache");
 const provider = new ethers.providers.Web3Provider(ganache.provider());
 ```
 
+### Browser Use
+
+You can also use Ganache in the browser by adding the following script to your HTML:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/ganache@version"></script>
```
Comment (MicaiahReid) [inline src/packages/ganache/README.md]:
```suggestion
<script src="https://cdn.jsdelivr.net/npm/ganache@{VERSION}/dist/web/ganache.min.js"></script>
```

**Pull request merged**